.github/workflows/release-swift-bindings-nix.yml

```
@@ -0,0 +1,107 @@
+      - name: Build archive
+        working-directory: bindings_ffi
+        run: |
+          mkdir -p Sources/LibXMTP
+          mv build/swift/xmtpv3.swift Sources/LibXMTP/
+          make framework
+          cp ../LICENSE ./LICENSE
+          zip -r LibXMTPSwiftFFI.zip Sources LibXMTPSwiftFFI.xcframework LICENSE
+      - name: Calculate checksum
+        id: checksum
+        working-directory: bindings_ffi
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:78: Double quote to prevent globbing and word splitting

(shellcheck)